test/integration/targets/ios_vlan/tests/cli/basic.yaml

```diff
@@ -0,0 +1,152 @@
+#---
```

Comment:
Whats the reason to comment out all the test cases?

Reply:
We do not have ios l2 switch image(15.2) to run on DCI. This is why we have to disable the ios_vlan test suite for now.